base/reduce.jl

```diff
@@ -396,7 +396,11 @@ end
 
 
 ## prod
+"""
+    prod(f, iter)
```

Comment:
looks like this'll need to be distinguished by signature in the stdlib doc index http://docs.julialang.org/en/stable/stdlib/collections.html?highlight=prod#Base.prod